.gitignore

```diff
@@ -1,3 +1,4 @@
 .dart_tool
+.idea/
```

Comment:
No! `~/.gitignore_global`! 😄

Reply:
I'm landing this as is Kevin, but am enclosing this gift to make up for it: 🎁